governance.md

```diff
@@ -0,0 +1,164 @@
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
+The purpose of a SIG is to own and develop a set of subprojects.
+
+A SIG can have its own policy for contribution, described in a README or CONTRIBUTING file in the SIG folder in this repo (e.g. sig-community/CONTRIBUTING.md), and its own mailing list, chat channel, etc.
+
+#### Governance 1.1 future groups
+[These descriptions](https://github.com/kubernetes/community#governance) are borrowed from [Kubernetes governance around community groups](https://github.com/kubernetes/community/blob/master/governance.md#community-groups).
+They are good-enough placeholders for discussions.
+
+* **Committees** are named sets of people that are chartered to take on sensitive topics. This group is encouraged to be as open as possible while achieving its mission but, because of the nature of the topics discussed, private communications are allowed. Examples of committees include the Technical Steering Committee or an overarching Steering Committee, and things like security or code of conduct.
+* **Subprojects** Each SIG can have a set of subprojects. These are smaller groups that can work independently. Some subprojects will be part of the main Kubernetes deliverables while others will be more speculative and live in the kubernetes-sigs github org.
+* **Working Groups** are temporary groups that are formed to address issues that cross SIG boundaries. Working groups do not own any code or other long term artifacts. Working groups can report back and act through involved SIGs.
+* **User Groups** are groups for facilitating communication and discovery of information related to topics that have long term relevance to large groups of Kubernetes users. They do not have ownership of parts of the Kubernetes code base.
+See the full governance doc for more details on these groups.
+
+### Decision making
+
+While the project is in the early stages, we are favoring a balance of including all voices and keeping a rapid pace of progress.
+To accomplish this balance, we are using a _consensus decision model_ that gives time and space for everyone to voice concerns with the proposal.
+
+There are two types of consensus models we use, the lightweight and the full:
+
+* **Lightweight consensus** occurs when a decision does not have wide or sweeping effects and is considered approved if no one raises any blocking objections after several days.
+In practice, an email is sent to a SIG or main community list, and if no one raises an objection after three days, the proposal passes.
+* **Full consensus** decisions require 3 yes votes (+1) and no objections (-1’s) and votes should be left open for at least 72 hours.
```

Comment:
Misread the "at least".
+1 to "at least 72 hours"

Reply:
weekend is problematic. We don't want to force folks to watch op1st on weekends to make sure they don't miss anything.

Reply:
Thanks all, I agree these are important clarifications to make. Do these changes seem more fair and equitable?

```suggestion
* **Full consensus** decisions require 3 yes votes (+1) and no objections (-1’s) and votes should be left open for at least 72 hours (not counting weekend days) to ensure all voices are heard.
72 hours is a minimum and should be increased for decisions with a likely broad impact, that involve contentious issues, or for any other reason that increases fairness in reaching consensus.
Specifically, these rules must **not** be used to intentionally or accidentally disenfranchise contributors.
For example, if a 72 hour window goes over a weekend and means a significant decision closes on a Monday, it may disenfranchise contributors who cannot watch the project daily.
In such cases, a full calendar week is a fair minimum window for decision making, balancing the needs to give people enough time and to keep the process moving along.
```